scripts/check_license.sh

```diff
@@ -77,17 +100,48 @@ for FILE in $VERSIONED_GO_FILES; do
     fi
 done
 
+# non-go files that include a license header
+for FILE in "${EXTRA_FILES[@]}"; do
+    if ! grep -qs "Copyright (C) 2019-$CURRENT_YEAR Algorand, Inc." "$PROJECT_ROOT/$FILE"; then
+        RETURN_VALUE=1
```

Comment:
minor: it seems like it should not set the return value to 1 if this is updating the files?

Reply:
As currently structured, it returns 1 if anything changes. I suppose a slight improvement might be to change all updates and in place edits to not set this and only print if $MOD_COUNT > 0, but I'm not sure we gain much by it.

Reply:
Ok, I didn't realize that. This makes sense